### DISCUSSION TITLE
b站推送没反应

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#277] [问题求助]
我搞错什么了吗（
![image](https://user-images.githubusercontent.com/66856838/146493553-7e5f00eb-4666-4c11-94af-bda5706213a9.png)
![image](https://user-images.githubusercontent.com/66856838/146493571-422b549e-ccaa-4e3f-9ae3-9d14e98c00d1.png)
59442895是uid，612645549是群号

**Comment by Tsuk1ko** [comment]
没写错，等有新动态或者开直播才会推送

（测试 log 忘删了

有问题的话看看 error log

**Reply by Miuzarte** [reply]
这是我的b站,发了五六条了没一次推送,好怪

**Reply by Tsuk1ko** [reply]
不过好像是有点问题，按理来说第二个不应该是 undefined

https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/37a46688da3ef68137f8e5d3338d40671bf50fdc/src/plugin/bilibili/dynamic.js#L120-L135

**Reply by Miuzarte** [reply]
> 不过好像是有点问题，按理来说第二个不应该是 undefined
> 
> https://github.com/Tsuk1ko/cq-picsearcher-bot/blob/37a46688da3ef68137f8e5d3338d40671bf50fdc/src/plugin/bilibili/dynamic.js#L120-L135

我刚刚转发了个动态然后现在推送过来了（
挺怪的
